gwu-libraries/sfm-twitter-harvester
# T1103 exporter v2

#### Summary

Updates harvester and exporter for Twitter v. 2 search and timeline types. Test in conjunction with `twitter-v2` branch in `sfm-ui` and `t1103-twitter-v2-search` branch in `sfm-utils`. (Note that this PR includes code both for the v2. harvester and the v2. exporter.)

#### To prepare

1. Update `docker-compose.yml` to use the v2 Twitter rest exporter as well as the existing exporter. See `example.docker-compose.yml` in the `t1103-twitter2-exporter` branch of `sfm-docker`. 
2. Change the compose file to build the local image for **both** the `twitterrestexporter` and the `twitterrestexporter2` services. 
3. Make sure both services are using code from the locally mapped volumes.

#### To test

1. Create collections for the Twitter 2 search and Twitter 2 user timeline harvest types.
2. Make sure collections harvest without errors.
3. Create exports of all types from these collections.
4. Create exports of all types from Twitter v1 search and Twitter v1 user timeline collections.
